src/skmultiflow/trees/nodes/lc_active_learning_node.py

```diff
@@ -0,0 +1,33 @@
+from skmultiflow.trees.attribute_observer import NumericAttributeClassObserverGaussian
+from skmultiflow.trees.attribute_observer import NominalAttributeClassObserver
+from skmultiflow.trees.nodes import ActiveLearningNode
+
+NAIVE_BAYES_ADAPTIVE = 'nba'
+
+
+class LCActiveLearningNode(ActiveLearningNode):
```

Comment:
Please add a short description.

Reply:
Description added in commit [5f331dc](https://github.com/scikit-multiflow/scikit-multiflow/pull/149/commits/5f331dce48293665f7ef78b059ba2d6782606438).